lms/envs/common.py

```diff
@@ -2215,4 +2215,4 @@
 ECOMMERCE_API_TIMEOUT = 5
 
 # Reverification checkpoint name pattern
-CHECKPOINT_PATTERN = r'(?P<checkpoint_name>\w+)'
+CHECKPOINT_PATTERN = r'(?P<checkpoint_name>[a-zA-Z0-9_ ]+)'
```

Comment:
This restriction will need to be enforced by the re-verification block once you add the author editing view for the checkpoint name.